src/components/Images.jsx

```diff
@@ -6,20 +6,63 @@ export default class Images extends React.Component {
     super(props);
 
     this.state = {
-
+      attractionImages: [],
+      displayedImage: '',
+      index: 0,
     };
+    this.changeImage = this.changeImage.bind(this);
   }
 
-  render() {
+  componentDidMount() {
     const { images } = this.props;
+    const sliced = images.slice();
+    this.setState({
+      attractionImages: sliced,
+      displayedImage: sliced[0],
+    });
+  }
+
+  changeImage(direction) {
```

Comment:
your implementation of the prev and next buttons here to change state are clear and concise- good job!